automation/scripts/testnet-keys.py

```diff
@@ -13,7 +13,7 @@
 client = docker.from_env()
 p = inflect.engine()
 
-CODA_DAEMON_IMAGE = "codaprotocol/coda-daemon:0.0.14-rosetta-scaffold-inversion-489d898"
+CODA_DAEMON_IMAGE = "gcr.io/o1labs-192920/coda-daemon-baked:0.2.6-5c08d6d-5c08d6d-testworld-2258826"
```

Comment:
This does not need a baked image, should maybe be refactored for `minaprotocol/mina-generate-keypair:0.2.9-a940247` but that requires more changes (the script would have to call `mina-generate-keypair` binary instead of `coda advanced generate-keypair`). At a minimum use `gcr.io/o1labs-192920/coda-daemon:0.2.6-5c08d6d`.